pkg/soperator/soperator_resources.go

```diff
@@ -359,16 +283,10 @@ func (specConfig *SpecConfig) getOperatorClusterRole() *horizoncomponents.Cluste
 		Resources: []string{"rolebindings", "roles"},
 	})
 
-	synopsysOperatorClusterRole.AddPolicyRule(horizonapi.PolicyRuleConfig{
-		Verbs:     []string{"get", "list", "watch", "create", "update", "patch", "delete", "deletecollection"},
-		APIGroups: []string{"batch", "extensions"},
-		Resources: []string{"jobs", "cronjobs"},
-	})
-
 	synopsysOperatorClusterRole.AddPolicyRule(horizonapi.PolicyRuleConfig{
 		Verbs:     []string{"get", "list", "watch", "create", "update", "patch", "delete", "deletecollection"},
 		APIGroups: []string{"extensions", "apps"},
-		Resources: []string{"deployments", "deployments/scale", "deployments/rollback", "statefulsets", "statefulsets/scale", "replicasets", "replicasets/scale", "daemonsets"},
+		Resources: []string{"deployments", "deployments/scale", "deployments/rollback"},
```

Comment:
Can you address TODO notes from here: https://github.com/blackducksoftware/synopsys-operator/pull/1080/files?  

If we're gonna clean up the RBAC, let's clean it all 😄

Reply:
added TODO back.. But removed something which I knew